test.js

```diff
@@ -204,17 +204,15 @@ test('opts.stdout:ignore - stdout will not collect data', async t => {
 
 test('helpful error trying to provide an input stream in sync mode', t => {
 	t.throws(
-		() => {
-			execa.sync('stdin', {input: new stream.PassThrough()});
-		},
+		() => execa.sync('stdin', {input: new stream.PassThrough()}),
```

Comment:
This was intentional. I don't like using inline arrow functions when the return value is not used as it makes the code less readable.

Reply:
Yes you're right, this is stricter